hubble/src/indexer/mod.rs

```diff
@@ -29,9 +29,33 @@ pub struct Indexer<T: FetcherClient> {
     pub indexer_id: IndexerId,
     pub start_height: BlockHeight,
     pub chunk_size: usize,
+    pub finalizer_config: FinalizerConfig,
     pub context: T::Context,
 }
 
+#[derive(Clone, Debug, serde::Deserialize)]
+pub struct FinalizerConfig {
+    // how many blocks to wait until a block is considered finalized (ie. there should be no reorgs).
+    // compensates for height differences between rpcs
+    // default: 5
+    #[serde(default)]
+    pub delay_blocks: usize,
+    // reload all block data after a block is considered finalized
+    // compensates for rpcs returning inconsistent results for non-finalized blocks.
+    // default: true
+    #[serde(default)]
+    pub reload: bool,
```

Comment:
this will default to false, requires the same as my previous comment

Reply:
done